packages/driver/src/cy/big_value.ts

```diff
@@ -0,0 +1,149 @@
+import _ from 'lodash'
+import $dom from '../dom'
+
+const LOGGED_ARRAY_SIZE = 10
+const LOGGED_OBJ_SIZE = 10
+const LONG_STRING_LENGTH = 100
+const MESSAGE_STRING_MAX = 30
+
+interface Values {
+  subject: any
+  expected: any
+  actual: any
+}
+
+interface Args {
+  0: boolean
+  1?: string
+  2?: string
+}
+
+export const replaceToBigValueTags = (args: Args, { subject, expected, actual }: Values) => {
+  if (typeof args[1] !== 'string' || typeof args[2] !== 'string') {
+    return args
+  }
+
+  if (isBigValue(subject)) {
+    args = replaceArgs(args, /#{this}/g, '%{this}')
+  }
+
+  if (isBigValue(expected)) {
+    args = replaceArgs(args, /#{exp}/g, '%{exp}')
+  }
+
+  if (isBigValue(actual)) {
+    args = replaceArgs(args, /#{act}/g, '%{act}')
+  }
+
+  return args
+}
+
+export const logBigValue = (value: any) => {
```

Comment:
this shouldn't be named `log*` since it's not calling into console.log or printing anything. its formatting the value for display

Reply:
How about changing it to `formatBigValue`?